src/components/tooltip/styles/STooltip.tsx

```diff
@@ -3,22 +3,22 @@ import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from "../../../consta
 
 interface ITooltip {
   closing: boolean;
-  color?: string;
+  backgroundColor: string;
+  color: string;
 }
 
 const Bronze = css<ITooltip>`
   display: flex;
   z-index: 120;
   width: max-content;
 
-  font-family: Inter;
-  font-size: 12px;
-  line-height: 1.67;
-  font-weight: 600;
-  border-radius: 6px;
-  background-color: ${(props) => (props.color ? props.color : "var(--neutral-600)")};
-  color: var(--ui-01);
+  border-radius: 4px;
+  background-color: ${(props) => props.backgroundColor};
+  color: ${(props) => props.color};
   animation: ${(props) => (props.closing ? "close" : "open")} 100ms ease-in-out;
```

Comment:
@nicolasshiken Testing the black mode you will notice that there is a timing difference between the body of the tooltip and its tip. Please make them work in synchronization, to appear and disappear at the same time.

Reply:
Yes I noticed that! But I think that is related to the Popover component, so maybe I can fix that on another PR?

Reply:
Yeah, that works too, will create a new ticket.